autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -138,6 +138,18 @@ private DistroXAutoscaleClusterResponse createClusterJsonResponse(Cluster cluste
         return distroXAutoscaleClusterResponseConverter.convert(cluster);
     }
 
+    private DistroXAutoscaleClusterResponse updateClusterAutoScaleState(Cluster cluster, AutoscaleClusterState autoscaleState) {
+        // TODO CB-15139 Use alertValidator to check for StopStart scaling entitlement
```

Comment:
Could you please file this as a new jira so that it doesn't get lost as a generic TODO

Reply:
Yes, filed as a separate JIRA and updated TODO comment: [CB-15673](https://jira.cloudera.com/browse/CB-15673)